sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py

```diff
@@ -302,10 +422,13 @@ def begin_recognize_content(self, form, **kwargs):
         # type: (Union[bytes, IO[bytes]], Any) -> LROPoller[List[FormPage]]
         """Extract text and content/layout information from a given document.
         The input document must be of one of the supported content types - 'application/pdf',
-        'image/jpeg', 'image/png' or 'image/tiff'.
+        'image/jpeg', 'image/png', 'image/tiff' or 'image/bmp'.
 
-        :param form: JPEG, PNG, PDF and TIFF type file stream or bytes.
+        :param form: JPEG, PNG, PDF, TIFF, or BMP type file stream or bytes.
         :type form: bytes or IO[bytes]
+        :keyword list[str] pages: Custom page numbers for multi-page documents(PDF/TIFF), input the number of the
```

Comment:
should we add an example of how to do it? or tmi?

Reply:
I thought about that, but seeing as we're just exposing the generated code as of right now, I kind of would rather wait to showcase those pieces in samples until we actually design for it next preview. what do you think?

Reply:
ahh good point. yeah we can wait